### ISSUE TITLE
--mixall flag in start.sh

### ISSUE THREAD

**Issue by FrancisPouliot** [schulterklopfer/wasabi_docker#1]
WalletWasabi.Gui.dll mix --wallet:testWallet --mixall --keepalive

I think having mixall could be a potential problem, since that will keep mixing coins that are already mixed. not entirely sure how it works though

**Comment by nopara73** [comment]
Yes, that's exactly how it works. If you specify `--mixall` then it'll cycle through the mix every coin, even if a coin's anonymity set is over 9000.

**Comment by schulterklopfer** [comment]
@FrancisPouliot: Well I could make things configurable through the environment. What kinda problems do you see?
